src/app/globals.css

```
@@ -6,19 +8,17 @@
 
 :root {
   /* Colors */
-  --color-primary: #68fe9c;
-  --color-secondary: #e2e2e2;
-  --color-shade-first: #060606;
-  --color-shade-second: #1b1f24;
-  --color-shade-third: #23272c;
-  --color-shade-fourth: #393d43;
-  --color-shade-fifth: #585d64;
-  --color-alert: #b61313;
+  --color-primary: #2b2b2d;
+  --color-highlight: #fe804b;
+  --gradient-primary: linear-gradient(#fe804b, #fb31b3);
```

We might need this gradient in SVG files. Can you split this into multiple variables?

```suggestion
  --gradient-primary-from: #fe804b;
  --gradient-primary-to: #fb31b3;
  --gradient-primary: linear-gradient(var(--gradient-primary-from), var(--gradient-primary-to));
```